node/runtime/src/lib.rs

```diff
@@ -66,8 +66,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node"),
 	impl_name: create_runtime_str!("substrate-node"),
 	authoring_version: 10,
-	spec_version: 16,
-	impl_version: 16,
+	spec_version: 17,
```

Comment:
Just something to keep an eye on, it was bumped to 17 here - https://github.com/paritytech/substrate/pull/1591/files#diff-5e5e1c3aec9ddfde0a9054d062ab3db9R97 - so depending on which one goes in first, we may need an adjustment.

Reply:
And in #1619 ... but we really should only need one bump per release at most.

Reply:
Once people stop using `master` branch on public testnets, sure.